index.html

```diff
@@ -0,0 +1,1367 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <!-- required meta tags -->
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <!-- bootstrap css etc -->
+  <link rel="stylesheet" type="text/css" href="assets/reset.css" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="assets/style.css" />
+  <title>Newsweek Clone</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+    integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous" />
+  <link rel="shortcut icon" href="http://s.newsweek.com/sites/www.newsweek.com/themes/newsweek/favicon.ico"
+    type="image/vnd.microsoft.icon" />
+</head>
+
+
+<header>
+  <!-- navbar fixed lg and xl screen-->
+  <nav class=" sticky-top d-none d-lg-flex flex-column d-sm-none p-2 mb-2">
+      <div id="top-nav" class="top-nav d-flex flex-row justify-content-between align-items-end pb-3 square-red">
+        <div id="date" class="date text-white pl-4">
+          <p>Wed, Sep 11, 2019</p>
+        </div>
+        <div id="top-logo" class="top-logo c-pointer">
+          <img src="assets/images/test-logo.jpg">
```

Comment:
`<img>` tags should contain the alt attribute which gives a short description for the picture. More [here](https://validator.w3.org).